src/index.html

```diff
@@ -11,12 +11,57 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
+    <header class="header">
+      <a class="logo__link">
```

Comment:
The `<a>` tag for the logo lacks an `href` attribute. This attribute is necessary for the link to function properly. Consider adding a valid URL or `#` if it's meant to be a placeholder.